server.js

```diff
@@ -253,7 +254,7 @@ app.post('*', function (req, res, next) {
     url: config.asr_url,
     method: 'POST',
     body: opusdec.stdout,
-    headers: {'Content-Type': 'application/octet-stream'},
+    headers: {'Content-Type': 'application/octet-stream', 'Language': req.headers.language},
```

Comment:
Should validate this too and test that 1) invalid value languages are rejected and 2) add a valid language to the happy/success path

Reply:
I think this validation should be done in the speech server itself, instead here, which is only the middleware. I'll prepare a PR for the speech server and share with you

Reply:
We are already defaulting to english here: https://github.com/mozilla/gspeech/commit/6367878e1950909c017b12cdd8b85a51574edb04  but I think a white list is important even though I have some concerns, but I prefer to have that and discuss in the speech-to-text server.